pages/profile/components/TableList.js

```diff
@@ -3,25 +3,24 @@ import React from "react";
 var i = 0;
 
 const TableList = (props) => {
+  var detailsArray = props.allDetails?.reverse();
   return (
     <table className={`table table-striped ${props.tableclass}`}>
       <thead>
         <tr>
           <th scope="col">{props.flat}</th>
           <th scope="col">{props.loc}</th>
-          <th scope="col">{props.siteName}</th>
           <th scope="col">{props.available}</th>
           <th scope="col">{props.view}</th>
         </tr>
       </thead>
       <tbody>
-        {props.tableData?.map((data) => {
+        {detailsArray?.map((data) => {
```

Comment:
add if condition that all the selected sites are only visible in the table

Reply:
Done sir